HdmiInput/HdmiInput.cpp

```diff
@@ -164,11 +172,11 @@ namespace WPEFramework
             int portId = 0;
             try {
                 portId = stoi(sPortId);
-            }catch (const device::Exception& err) {
-                LOG_DEVICE_EXCEPTION1(sPortId);
-                returnResponse(false);
+            }catch (...) {
+		    LOGWARN("Invalid Arguments");
+		    response["message"] = "Invalid Arguments";
```

Comment:
can you please avoid adding undocumented things to the JSON-RPC interface. This is your interface - HdmiInput.json. It does not have "message". According to JSON-RPC protocol https://www.jsonrpc.org/specification#response_object result object does not exist in case of error. I.e. no body will see your "message" anyway

Reply:
You can add "message" to the error object instead of result - https://www.jsonrpc.org/specification#error_object

Reply:
@anand-ky error object is internally set by Thunder to reflect the error code returned by the API. Example of error codes: https://github.com/rdkcentral/Thunder/blob/82d4791478934292f7260fef233b9743fa34d88e/Source/core/Portability.h#L670 . If people want to document a description of the error they should use schema, like here:  https://github.com/rdkcentral/rdkservices/blob/b83fe9cb6cf6c5535f82031f24865a96e3c4b61a/Messenger/Messenger.json#L74

Reply:
Addressed in the latest patch.